### ISSUE TITLE
Network Connectivity: IP and MAC Address (App)

### ISSUE THREAD

**Issue by umbhau** [Opentrons/opentrons#2181]
As a robot user, I would like to know my robot's IP and MAC addresses, so that I can use them (among other things) to authenticate to networks.

## Acceptance Criteria
- User able to see their robot's IP and MAC in the Run App

## Implementation Notes
- Dependent on #2136

**Comment by umbhau** [comment]
Closing in favor of #2346
